react/details/src/components/entity-field-table/EntityFieldTable.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import EntityField from 'components/entity-field-table/EntityField';
+
+import { entityFieldsType } from 'components/_types/entity';
+
+const EntityFieldTable = ({ entity }) => {
+  const { t } = useTranslation();
```

Comment:
Hooks are cool, but I'd like to use them always or never in our generated widgets implementation. We could start without them and then improve our code generator to generate widgets with hooks.